common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -628,6 +635,10 @@ div.problem {
       height: ($baseline*2);
       vertical-align: middle;
       font-weight: 600;
+
+      @media print {
+        display: none;
```

Comment:
Thanks for the note here, that makes sense to me. As you noted these input controls don't change the content / setup of the problem.